Igalvan - Livingston

```diff
@@ -0,0 +1,3 @@
+Livingston
```

Comment:
Hi, it's desert area, so really hot in the summer and sort of cold in the winter, it doesn't snow but it hails and the grass freezes to a level where you can glide with your shoes, kind of fun. You can see all the seasons there, fall is really nice cause the leaves on the trees turn colors  of red, yellow, orange...